<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roomPlayerform</class>
 <widget class="QWidget" name="roomPlayerform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>99</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_identify">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_badge">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ä¸Š</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_num">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_icon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_edge">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>pb_edge</zorder>
  <zorder>pb_icon</zorder>
  <zorder>lb_identify</zorder>
  <zorder>lb_badge</zorder>
  <zorder>lb_num</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
